charts/redpanda/templates/statefulset.yaml

```diff
@@ -295,16 +297,14 @@ spec:
             periodSeconds: {{ .Values.statefulset.readinessProbe.periodSeconds }}
             successThreshold: {{ .Values.statefulset.readinessProbe.initialDelaySeconds }}
           command:
-            - bash
-            - -c
-            - |
-              rpk redpanda start \
-                --smp={{ include "redpanda-smp" . }} \
-                --memory={{ template "redpanda-memory" . }}M \
-                --reserve-memory={{ template "redpanda-reserve-memory" . }}M \
-                --default-log-level={{ .Values.logging.logLevel }} \
-                --advertise-rpc-addr={{ $internalAdvertiseAddress }}:{{ .Values.listeners.rpc.port }} |
-              sed 's@RP_BOOTSTRAP_USER[^ ]\+@<censored>@g'
+            - rpk
```

Comment:
wow, i keep forgetting this is a thing. Glad we can move away from this. There are other places we are still using bash, we may want to consider reviewing those also.